addons/makeslave.pm

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/perl
+
+use Config::IniFiles;
+
+my %ini;
+
+tie %ini, 'Config::IniFiles';
+
+my $pfconf = "/usr/local/pf/conf/pf.conf";
+
+my %inipfconf;
+tie %inipfconf, 'Config::IniFiles', ( -file => "$pfconf" );
+
+my $replication_username = $inipfconf{"active_active"}{"galera_replication_username"};
+my $replication_password = $inipfconf{"active_active"}{"galera_replication_password"};
+
+
+print "Enter the MySQL root password: ";
+my $mysql_root_password = <STDIN>;
+chomp $mysql_root_password;
+
+
+print "Enter the MySQL master ip address: ";
+my $mysql_master_ip = <STDIN>;
+chomp $mysql_master_ip;
```

Comment:
Add validation of the IP address to make sure the user doesn't enter something like `192.168.1.myballs`